torchscale/architecture/config.py

```diff
@@ -219,7 +219,7 @@ def __init__(self, **kwargs):
         super().__init__(**kwargs)
         self.decoder_embed_dim = kwargs.pop("decoder_embed_dim", 768)
         self.decoder_value_embed_dim = kwargs.pop("decoder_value_embed_dim", 1280)
-        self.decoder_retention_heads = kwargs.pop("decoder_retention_heads", 3)
+        self.decoder_retention_heads = kwargs.pop("decoder_retention_heads", 4)
```

Comment:
@KimballNJardine @nprisbrey I propose that we change this one default value from 3 to 4 because it does not actually run inference successfully with the other defaults. (I think because of the `embed_dims % (head *2)`)